### PR TITLE
Add pr merge docs

### PR DESCRIPTION
**Pull request by AgustinRamiroDiaz**
After talking with @teresaromero we found that we had different criteria for merging PRs. 

This PR is an attempt to write down how we'd like out usual PR-merging to be, and also to get more feedback on the different views of the team

### CODE REVIEW

File: docs/code-review.md [hunk 1]
```diff
@@ -62,6 +63,10 @@ This approach can be summarized by the term **Minimum Viable Change (MVC)**. MVC
 
 If the developer decides to open a new issue from the current one or is not completely sure whether it is necessary to create a new issue or not, this must be commented in the original issue/PR.
 
+### PR Merge
+
+Once the PR is approved and the checks are passing, the author will be responsible for merging the PR or delegating the responsibility to another developer. There could be exceptions to this rule, for example, if the author is on vacation and the merge is "urgent", other developers can merge the PR to unblock the situation.
```
Comment (jLopezbarb) [inline docs/code-review.md:68]: Can we add what happen if the author is an external contributor? They are not allowed to merge the PR

Comment (AgustinRamiroDiaz) [reply docs/code-review.md:68]: Sure! How do you think we should handle that situation?

I'm thinking that  
- If there are maintainers among the reviewer, they should merge the pr as they have better context
- If there are not maintainers among the reviewers, they would need to wait for a maintainer to take responsibility. They could ping the maintainers to remind them of the pr

Comment (jLopezbarb) [reply docs/code-review.md:68]: Maintainers are the only ones that can merge PRs. What I would do is:

- when all the requests from the maintainers have been completed(all the maintainers that have written something accepts the PR), merge
- I would ask for two approvals when an external contributor makes a PR, just to ensure that the PR is aligned with what we want

**Review by jLopezbarb — APPROVED** [review]

File: docs/code-review.md [hunk 2]
```diff
@@ -62,6 +63,19 @@ This approach can be summarized by the term **Minimum Viable Change (MVC)**. MVC
...
+#### **If the author is a mantainer**
+
+Once the PR is approved and the checks are passing, the author will be responsible for merging the PR or delegating the responsibility to another developer. There could be exceptions to this rule, for example, if the author is on vacation and the merge is "urgent", other developers can merge the PR to unblock the situation.
+
+#### **If the author is not a mantainer**
+
+Any mantainer can merge the PR when:
+
+- The PR has approvals from at least two maintainers to ensure that the changes are aligned with the project's goals and quality standards.
+- All requests from the mantainers have been addressed, and those mantainers have approved the PR.
```
Comment (jLopezbarb) [inline docs/code-review.md:77]:
```suggestion
- All requests from the maintainers have been addressed, and those maintainers have approved the PR.
```

File: docs/code-review.md [hunk 3]
```diff
@@ -62,6 +63,19 @@ This approach can be summarized by the term **Minimum Viable Change (MVC)**. MVC
 
 If the developer decides to open a new issue from the current one or is not completely sure whether it is necessary to create a new issue or not, this must be commented in the original issue/PR.
 
+### PR Merge
+
+#### **If the author is a mantainer**
+
+Once the PR is approved and the checks are passing, the author will be responsible for merging the PR or delegating the responsibility to another developer. There could be exceptions to this rule, for example, if the author is on vacation and the merge is "urgent", other developers can merge the PR to unblock the situation.
+
+#### **If the author is not a mantainer**
```
Comment (jLopezbarb) [inline docs/code-review.md:72]:
```suggestion
#### **If the author is not a maintainer**
```

File: docs/code-review.md [hunk 4]
```diff
@@ -62,6 +63,19 @@ This approach can be summarized by the term **Minimum Viable Change (MVC)**. MVC
 
 If the developer decides to open a new issue from the current one or is not completely sure whether it is necessary to create a new issue or not, this must be commented in the original issue/PR.
 
+### PR Merge
+
+#### **If the author is a mantainer**
```
Comment (jLopezbarb) [inline docs/code-review.md:68]:
```suggestion
#### **If the author is a maintainer**
```

**Review by ifbyol — APPROVED** [review]

**Comment by jLopezbarb** [comment]
Check pre-commit linter before merging but LGTM!

**Pull request merged**